### ISSUE TITLE
"Authentication failed: "no body"" - on automatic authentication

### ISSUE THREAD

**Issue by eswiking** [586837r/node-red-contrib-alexa-remote2#234]
hello,
i am trying to establish automatic authentication with my amazon accuount.
i have defined file path /opt/iobroker/amazon.txt
i see that the {"loginCookie":"session-id= ...} was written into the file after successfull authentication via web-webbrowser.
but when i deploy the flow in node-red again i get that error:
"Authentication failed: "no body""

is there anything e.g. workaround to fix this and get automatic authentication working ?

**Comment by arwindersingh82** [comment]
i'm getting the same issue - any work around or fix?

**Comment by eswiking** [comment]
no, no fix for that issue yet. but as i found out, i do not need it. i can access all avalible alexa functions via the iobroker object tree, as switching smarthome devices, let alexa speak, playing radio, etc. 
mabe  the fist manual authentication was enough for make it work.
